docs/releases/5.2.txt

```diff
@@ -41,6 +41,17 @@ Minor features
   :ref:`extrabody <extrabody>` for adding custom code before the closing
   ``</body>`` tag.
 
+* In order to improve accessibility:
+
+  * Form fields in the change form are now shown below their respective labels
+    instead of next to them.
+
+  * Help text in the change form is now shown after the field label and before
+    the field input.
+
+  * Validation errors in the change form are now shown after the help text and
+    before the field input.
```

Comment:
```suggestion
* In order to improve accessibility of the admin change forms:

  * Form fields are now shown below their respective labels instead of next to
    them.

  * Help text is now shown after the field label and before the field input.

  * Validation errors are now shown after the help text and before the field
    input.
```
I wonder if this should be in the `Miscellaneous Backwards incompatible changes in 5.2` section as it isn't so much a feature but rather a change we wish to alert folks of :thinking: